src/ImmutableStore.es6.js

```diff
@@ -19,6 +19,9 @@ var invariant = require('invariant');
 var Immutable = require('immutable');
 var ObjectOrientedStore = require('./ObjectOrientedStore.es6');
 
+const IN_PRODUCTION = process.env.NODE_ENV === 'production';
+const CHECK_IMMUTABILITY = !!process.env.CHECK_IMMUTABILITY;
```

Comment:
we actually cant do this due to how our webpack replace plugin is set up.  only NODE_ENV will work.  I am fine just using that, I don't see anyone complaining about this.